edenai_apis/apis/openai/openai_audio_api.py

```
@@ -173,3 +175,114 @@ async def audio__atext_to_speech(
+        resolved_voice = voice or "alloy"
+        resolved_speed = normalize_speed_for_openai(speed)
+
+        payload = {
+            "model": resolved_model,
+            "input": text,
+            "voice": resolved_voice,
+            "speed": resolved_speed,
+            "response_format": audio_format or "mp3",
+        }
+
+        response = requests.post(url, json=payload, headers=self.headers)
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)